tempesta_fw/server.c

```diff
@@ -249,25 +229,14 @@ tfw_sg_release_all(void)
 	TfwServer *srv, *srv_tmp;
 	TfwSrvGroup *sg, *sg_tmp;
 
-	write_lock(&sg_lock);
```

Comment:
Please comment that we don't need the lock because the function is called at shutdown time, when there are no users any more.